### DISCUSSION TITLE
Linting media queries

### DISCUSSION THREAD

**Discussion by amorriscode** [vanilla-extract-css/vanilla-extract#480] [Q&A]
Howdy!

We've been using Vanilla Extract successfully in production for a while now. Today we hit a little snag.

We wrote an invalid media query and it wasn't caught by the linter. In the end, this meant a CSS file was generated that had invalid syntax.

When using an [online CSS validator](https://jigsaw.w3.org/css-validator/validator), this is what we noticed:

![image](https://user-images.githubusercontent.com/16005567/142326962-226ccd18-f373-48f9-938e-9e3d9f3f1176.png)

If you write a media query like this, you can recreate the issue:

```TypeScript
"@media": {
  "(max-width: 320px": { ... }
}
```

So it's clear that the strings _aren't_ being linted. This might not even be a Vanilla Extract issue ([there was a post on CSSType about media queries](https://github.com/frenic/csstype/issues/66)). Do you have any ideas on how something like this could be avoided? Is this a config issue on our end?

**Comment by mattcompiles** [comment]
You're right in that there's currently no validation for queries. Definitely open to adding this though.
